src/Adapter/Image/ImageGenerator.php

```diff
@@ -120,14 +122,24 @@ protected function resize(string $filePath, ImageType $imageType): bool
             $forceFormat = ($imageFormat !== 'jpg');
             if (!ImageManager::resize(
                 $filePath,
-                sprintf('%s-%s.%s', rtrim($filePath, '.' . $imageFormat), stripslashes($imageType->name), $imageFormat),
+                sprintf('%s-%s.%s', dirname($filePath) . DIRECTORY_SEPARATOR . $imageId, stripslashes($imageType->name), $imageFormat),
                 $imageType->width,
                 $imageType->height,
                 $imageFormat,
                 $forceFormat
             )) {
                 $result = false;
             }
+            if ($generate_high_dpi_images && !ImageManager::resize(
```

Comment:
This feature has been removed from develop branch, this whole block can be removed

Reply:
Yep, thanks @jolelievre